cli/command/utils.go

```diff
@@ -50,21 +46,14 @@ func CopyToFile(outfile string, r io.Reader) error {
 	return nil
 }
 
-var ErrPromptTerminated = errdefs.Cancelled(errors.New("prompt terminated"))
+const ErrPromptTerminated = prompt.ErrTerminated
```

Comment:
Just realised that this error is asserted in `internal/test`, which is outside of the `cli/`package, so won't have access to `cli/internal/....`


We should probably consider unifying the `internal/` and `cli/internal` packages if we need some of those to be accessible outside of `cli/`


https://github.com/docker/cli/blob/ff5fdfae357b1e8ab0ee7f3674c834aa5ea5404e/internal/test/cmd.go#L15

Reply:
Updated; moved it to the top-level `internal/` package instead of `cli/internal/`